.rubocop.yml

```diff
@@ -1,3 +1,8 @@
 inherit_gem:
   rubocop-govuk:
     - config/default.yml
+
+Metrics/BlockLength:
```

Comment:
Maybe this can wait for https://github.com/alphagov/rubocop-govuk/pull/65?

Reply:
Cool. I await a new release!